presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -59,6 +59,10 @@ statement
     | CREATE (OR REPLACE)? VIEW qualifiedName AS query                 #createView
     | DROP VIEW (IF EXISTS)? qualifiedName                             #dropView
     | CALL qualifiedName '(' (callArgument (',' callArgument)*)? ')'   #call
+    | CREATE ROLE name=identifier
+        (WITH ADMIN grantor)?
+        (IN catalog=identifier)?                                       #createRole
```

Comment:
I thought we agreed that the `IN` extension should not be supported. ROLE manipulations should be allowed only within the session catalog.

Reply:
I thought we agreed to merge this as it is and fix it later. Would you accept a fix on top of this? Instead of doing rebase one more time?

Reply:
> I thought we agreed to merge this as it is and fix it later. Would you accept a fix on top of this? Instead of doing rebase one more time?

That's what I also think was decided.

@ilfrin @martint you have more context.

Reply:
We decided to leave that functionality out since it's not clear exactly how it should work and there's an easy workaround.

Reply:
Sounds good. Would you accept a fix on top of this? Instead of doing rebase one more time?